packages/valaxy/node/plugins/valaxy.ts

```diff
@@ -19,6 +20,22 @@ import type { ValaxyNodeConfig } from '../types'
 import { checkMd } from '../markdown/check'
 import { vLogger } from '../logger'
 import { countPerformanceTime } from '../utils/performance'
+import { isProd } from '../utils/env'
+
+function generateConfig(options: ResolvedValaxyOptions) {
```

Comment:
For now, redirects is defined in siteConfig, so I wonder if we can get the configuration directly from the client without adding extra runtimeConfig.

Reply:
I guess we can save some code?
I am not directly debugging now, so you can tell me if i have any mistakes.

Reply:
When compiling, we need to collect redirect rules from frontmatter, whick are not user-defined. Isn’t it a good idea to put these in siteconfig?

Reply:
Since I lack experience, if you think this is better, I will modify it.

Reply:
OK, you are right, some configs collected from frontmatter.